internal/controller/dnsrecord_healthchecks_test.go

```diff
@@ -167,6 +167,64 @@ var _ = Describe("DNSRecordReconciler_HealthChecks", func() {
 		}, TestTimeoutMedium, time.Second).Should(Succeed())
 	})
 
+	It("Should create valid probe CRs with default values", func() {
+		//Create test dnsrecord with nils for optional fields
+		dnsRecord.Spec.HealthCheck = &v1alpha1.HealthCheckSpec{
+			Path: "/health",
```

Comment:
This is not the "minimal" health spec that we could have in the policy. We use the [same](https://github.com/Kuadrant/kuadrant-operator/blob/main/api/v1/dnspolicy_types.go#L56) spec for the policy and DNSRecord